frontend/src/components/pages/CreatorProfile/Carousel.tsx

```diff
@@ -0,0 +1,172 @@
+import "react-responsive-carousel/lib/styles/carousel.min.css";
+
+import { Flex, IconButton, Image } from "@chakra-ui/react";
+import { makeStyles } from "@material-ui/core";
+import React from "react";
+import { FcNext, FcPrevious } from "react-icons/fc";
+import { Carousel as ResponsiveCarousel } from "react-responsive-carousel";
+
+interface CarouselProps {
+  images: string[];
+}
+
+export type Presentations = {
+  name: string;
+  description: string;
+  gradeLevels: string[];
+  audienceSize: number;
+  locations: string[];
+  languages: string[];
+  avSetup: string;
+  remote: boolean;
+  saleOrSigning: boolean;
+  price: string;
+  mediaGallery: string[];
+};
+
+// SAMPLE DATA FOR TESTING ------------------------------
+export const presentationsTest: Presentations[] = [
```

Comment:
This should come from the server, and the correct way to add test data is by updating the seeding script and getting it from the server. Hardcoding this means we can't figure out any issues with the API until we actually call it. 

After looking into this, I did find quite a few issues with the creator presentations implementation on the backend actually, so we get to fix them as part of this. 


So, I'd recommend
- Getting rid of this and adding the presentations to the seeding data in `seeders/sample-creators-****` 
- Now, we need to fix the presentations field in the creator model. Right now, they point to another model, but that's obviously not the case since it's a JSON field. Delete `presentations.model.ts` and instead create an interface like `interface Presentation {... all the proprties of the presentation model .. }` 
- One fix you need to make is that `photos` should be an array of strings, not a string
- Now, you can run the seeder and get presentations into the database. May deal w some issues here, but then you can test with real data instead.